content/en/account_management/guide/hourly-usage-migration.md

```diff
@@ -316,11 +150,11 @@ ENDPOINT | PRODUCT FAMILY
 
 ## JSON:API Compliant Format
 
-Response bodies and parameter names conform to the [JSON:API specification][1]. All data
+Response bodies and parameter names conform to the [JSON:API specification][3]. All data
 available in the v1 APIs is still available. See the example below of the mapping from the v1 hosts
 API to the v2 hourly usage API.
 
-### V1 API: [Get hourly usage for hosts and containers][2]
+### V1 API: [Get hourly usage for hosts and containers][4]
```

Comment:
This was already here, but I don't think you can add a link to the header.

Reply:
gotcha - @maycmlee should we go ahead and remove it then?

Reply:
@katherinebroner that works for me, I wasn't sure if you wanted to add a link to it somewhere in the content below.

Reply:
thanks, will remove